CONTRIBUTING.md

```diff
@@ -163,13 +157,27 @@ firefox http://localhost:13000
 #### Start crawlers process
 
 ```ShellSession
-nix-shell --command monocle-api2-start
+nix-shell --command monocle-repl
 λ> import Macroscope.Worker
 λ> import Macroscope.Main
 λ> import Monocle.Client (withClient)
 λ> withClient "http://localhost:18080" Nothing $ \client -> runMacroscope 19001 "../etc/config.yaml" client
 ```
 
+#### Run ghcid
+
+```ShellSession
+nix-shell --command monocle-ghcid
+```
+
+#### Run all components in emac buffers
+
+Note that the commands below can be started in emacs buffers using:
+
+```ShellSession
+nix-shell --command launch-monocle-with-emacs
+```
+
```

Comment:
well, i think we can skip that command :-) 

```suggestion
```

Reply:
Done